src/seeder.js

```diff
@@ -72,22 +65,23 @@ export default class Seeder {
     /**
      * Await opening corestore and announcing the seeder on the seeder topic
      */
-    ready () {
+    ready() {
```

Comment:
that looks like editors settings are in conflict with linter

Reply:
Do you think adding boolean property `isReady` will be useful here?

Reply:
> that looks like editors settings are in conflict with linter
Will add Standard in another PR, because it will be too disruptive now for review.

>  Do you think adding boolean property isReady will be useful here?
Maybe, I can make `this._open()` return `true` and then `await seeder.ready()` would return true too. But I don't know where would I use that.